Objects/stringobject.c

```diff
@@ -1209,8 +1209,11 @@ string_richcompare(PyStringObject *a, PyStringObject *b, int op)
     PyObject *result;
 
     /* Make sure both arguments are strings. */
-    if (!(PyString_Check(a) && PyString_Check(b))) {
+     if (!(PyString_Check(a) && PyString_Check(b)) ) {
```

Comment:
I don't think the extra spaces are intended?

Reply:
Spaces removed